spec/ttml1.xml

```diff
@@ -6074,7 +6073,7 @@ the computed value, then the value least distant from 0, i.e., closest to the ba
 <note role="derivation">
 <p>The semantics of the style property
 represented by this attribute are based upon that defined by <bibref ref="xsl11"/>,
-&sect; 7.30.18.</p>
+&sect; 7.30.18, where the <el>tt</el> element is considered to establish the root stacking context.</p>
```

Comment:
This doesn't map to TTML2 where it would conflict with w3c/ttml2#470 and it also moves an effectively normative statement into an informative one. I'd suggest moving the statement to a separate `p` just before or after the one about treatment of the `auto` value.

(it's a separate question if we want to back-port the CSS mappings into TTML1 Third Ed)